packages/integrations/markdoc/src/content-entry-type.ts

```diff
@@ -232,7 +232,9 @@ function getStringifiedImports(
 		const resolvedPath =
 			config.type === 'local' ? new URL(config.path, root).pathname : config.path;
 
-		stringifiedComponentImports += `import ${importName} from ${JSON.stringify(resolvedPath)};\n`;
+		stringifiedComponentImports += `import ${importName} from ${JSON.stringify(
+			decodeURI(resolvedPath)
+		)};\n`;
```

Comment:
We can also fix this similarly by only changing the `new URL(config.path, root).pathname` to `fileUrlToPath(new URL(config.path, root))`